real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/simulation/SensorPlayer.java

```diff
@@ -122,163 +93,4 @@ else if (message.getLoggingMessage().equals(GuiLoggingButtonMessage.LoggingMessa
             }
         });
     }
-
```

Comment:
why did this get taken out? I'm putting it back in until we can get linebyline to work nicely